### DISCUSSION TITLE
make search bar size customizable

### DISCUSSION THREAD

**Discussion by jmlucjav** [espanso/espanso#1664] [Feature requests and Ideas]
Hi,

I have multiple matches with the same trigger, cause it is just easier to create them that way, and I don't need to come up with unique triggers (which would be difficult at some point).

These are programming related stuff, and they are long lines, many of which just differ in the end part of the line. Right now when I search for them, often I cannot see the differences between them, as I can only see the initial part of the line. Would it be possible to make the size configurable, or at least make the window resizable, and make it remember the preferred size? This would solve the issue

thanks

**Comment by jwigg-netscout** [comment]
I think that's a good suggestion 🙂 

In the mean time, while we wait for it to be implemented, have you tried using labels on your triggers: https://espanso.org/docs/matches/basics/#search-labels ?

**Comment by jmlucjav** [comment]
yeah I was aware there were 'labels', did not not look further as I hoping to have the match definition as short as possible. Anyway if there is no other way, I will have to resort to labels

thanks!
